### DISCUSSION TITLE
Release Nomenclature

### DISCUSSION THREAD

**Discussion by TwinkieXLII** [JGMaker3dofficial/artistd#25] [Ideas]
Proposal for Release nomenclature. I'm a big fan of year based release numbers. 

Year.Release.hotfix-marlinbase

2021.1-2.0.5. The First release of 2021, based from Marlin 2.0.5

2021.2.1-2.0.5. The Second Release of 2021 with hotfix 1.   Based on Marlin 2.0.5.

**Comment by mikejr83** [comment]
I'd prefer semantic version numbers. I don't mind doing them year based.

`2021.02.12` - Released the 12th of Feb, 2021. 
`2021.02.10-beta` - Beta pre-release that was released on the 10th.

They're always increasing by date so you know how old they are.

My personal opinion is that this firmware's version should be just that, this firmware's. The Marlin bootscreen flashes the Marlin version. It should be pretty obvious from just turning the printer on what version of Marlin the machine is running. I think just making it a point to leave a "Based on Marlin: v2.0.7.2" on the releases page would be enough.

**Reply by TwinkieXLII** [reply]
The way it is setup in the version.h files, the marlin logo will remain, but can be made to have the machine's firmware version splashed up on the screen.  I like the idea of adding the month, but I'm not sure we need the day.    The  We could leave the Marlin Version in the release pages and perhaps the info menu.
